ClosedXML/ClosedXML.csproj

```diff
@@ -16,36 +16,27 @@
 		<PackageReleaseNotes>$(LAST_COMMIT_MESSAGE)</PackageReleaseNotes>
 		<Description>ClosedXML is a .NET library for reading, manipulating and writing Excel 2007+ (.xlsx, .xlsm) files. It aims to provide an intuitive and user-friendly interface to dealing with the underlying OpenXML API.</Description>
 		<Copyright>MIT</Copyright>
+		<PackageIcon>Product.png</PackageIcon>
 		<PackageProjectUrl>https://github.com/stesee/ClosedXML</PackageProjectUrl>
-		<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
are there too many warnings at the moment or why is that turned off?

Reply:
All warnings were disabled using <NoWarn>. I fixed the probably accidential disabled output of warnings by removing <NoWarn>.